### ISSUE TITLE
Update Sourcing BOM from Beta CAD

### ISSUE THREAD

**Issue by ebhillstrom** [RespiraWorks/Ventilator#645]
(Once Beta CAD is complete)

**Comment by DonnaCG** [comment]
Does CoVent BOM close this task: https://drive.google.com/file/d/1cpdT3-IfY7Cz_9TQFog_xkggR2afI013/view

**Comment by martukas** [comment]
This is superseded by #820 and henceforth BOM will be managed with tighter version control.
